geopandas/io/parquet.py

```diff
@@ -0,0 +1,223 @@
+import json
+import warnings
+
+from pandas.io.common import get_filepath_or_buffer
+from pandas.io.parquet import PyArrowImpl as PandasPyArrowImpl
+from pandas import DataFrame
+
+from shapely.wkb import loads
+
+from geopandas import GeoDataFrame, GeoSeries
+
+
+def _encode_crs(crs):
+    if isinstance(crs, str):
+        return {"proj4": crs}
+
+    return crs
```

Comment:
I had some thoughts about the format of the CRS metadata the other day and I thought that in the interests of redundancy and interoperability it might be helpful for the "geoparquet" metadata standard to include three definitions or formats of the CRS: WKT, PROJ, and authority code (i.e. EPSG code). I had drafted up something like this:
```python
# capture CRS
if self.crs:
    crs = pyproj.CRS.from_user_input(self.crs)
else:
    crs = {}
# assign CRS data to new metadata if CRS exists
if crs:
    crs_val = {
        "wkt" : crs.to_wkt(),
        "proj" : crs.to_dict(),
        "auth" : {"auth_name" : "epsg", 
                  "auth_code" : crs.to_epsg(min_confidence=0)}
    }
    geometry_metadata["geometry_fields"][0]["crs"] = crs_val
```

PS thank you for all your work on this PR @brendan-ward 
PPS I'm not familiar with PR and code review process so kindly let me know if I should contribute my comments otherwise in future!